src/libs/nav.js

```diff
@@ -82,19 +85,6 @@ export const useRoute = () => {
   return parseRoute(handlers, location)
 }
 
-export const TitleManager = () => {
```

Comment:
Moved this component to avoid having nav import from brand-utils (which imports from nav). This circular import caused a test to fail.